llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import re
+
+def filter_definition (line, GOList, filterStr):
+  if any(f in line for f in GOList):
+    return re.sub(filterStr, '', line)
+  return line
+
+def remove_GO_ticketnode (GOList, isFunction, inputFile, outputFile):
+  """
+  Remove the GO's repo_ticket metadata.
+  """
+  filterStr = "!repo_ticket ![0-9]+"
+  if isFunction:
+    GOList = ["@{0}()".format(i) for i in GOList]
+  else:
+    GOList = ["@{0} =".format(i) for i in GOList]
```

Comment:
We’ve got two identical loops here (with the format string being the only difference). Could we eliminate the duplication?

Reply:
I have used `GOList = ["@{0}".format(i) for i in GOList]` for both functions and global variables. The function `filter_definition` is modified for handling both cases.